src/Resources/Guests/index.ts

```diff
@@ -2,17 +2,20 @@ import Client from '../../Client';
 import Baskets from './Baskets';
 import Gifts from './Gifts';
 import ShippingAddons from './ShippingAddons';
+import UndeliverableDate from './UndeliverableDate';
 
 export interface GuestsResources {
     baskets: Baskets;
     gifts: Gifts;
     shippingAddons: ShippingAddons;
+    undeliverableDates: UndeliverableDate;
```

Comment:
These don't belong to a guest (you wouldn't say that a guest user owns these resources), so I don't believe this is the correct place for this.

I imagine it should go into it's own resource, or it should be a method in both orders and gifts.

So either

`$whirli.undeliverableDates.all()`

or

`$whirli.orders.getUnderliverableDates()`
and
`$whirli.gifts.getUndeliverableDates()`


or

`$whirli.orders.undeliverableDates.all()`
and
`$whirli.gifts.undeliverableDates.all()`


Any of the above works for me, so if you're happy with any of them go for it, otherwise let me know and we can chat about it